<?xml version="1.0" encoding="UTF-8"?>
<launch>

<node name="lidar_process" pkg="projet" type="lidar_process.py"/>

<!-- <node name="camera_info" pkg="projet" type="camera_calib.py"/>

<param name="camera_calib" value="$(find projet)/calib/raspicam.yaml" />

<node pkg="image_proc" type="image_proc" name="image_proc" output="screen">
    <remap from="image_raw" to="camera/image"/>
    <remap from="camera_info" to="camera/camera_info_correct"/>
  </node> -->


<!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find projet/rviz/turtlebot.rviz"/> -->

</launch>
